assets/js/watch.js

```diff
@@ -219,7 +219,27 @@ function get_youtube_comments() {
         '&hl=' + video_data.preferences.locale +
         '&thin_mode=' + video_data.preferences.thin_mode;
 
-    var onNon200 = function (xhr) { comments.innerHTML = fallback; };
+    var onNon200 = function (xhr) {
+        if (!video_data.comments_enabled) {
+            comments.innerHTML = `
+            <div id="comments-turned-off-on-video-message" class="h-box v-box">
+                <p><b>${video_data.youtube_comments_disabled_text}</b></p>
+
+                <p>
+                    <b> 
+                        <a href="javascript:void(0)" data-comments="reddit" id="try-reddit-comments-link"> 
+                            ${video_data.youtube_comments_disabled_try_reddit}
+                        </a> 
+                    </b>
+                </p> 
+            </div>`;
```

Comment:
```suggestion
                <p><b><a href="javascript:void(0)" data-comments="reddit" id="try-reddit-comments-link">
                    ${video_data.youtube_comments_disabled_try_reddit}
                </a></b></p>
            </div>`;
```

Reply:
This can be condensed a bit ^^
Also, I'm not sure about `javacsript:void(0)` links (See #769)